### PR TITLE
Issue 171

### PR DESCRIPTION
**Pull request by kimpham54**
# What does this Pull Request do?
Related to https://github.com/digitalutsc/islandora_web_annotations/issues/171
Couldn't use Views PHP, also had to add views for particular content models. Did a final cleanup and updated the wiki for installation.

# What's new?
* Adds a View for Image, Page that renders markup
* Adds a View for Audio, Video that renders markup

# How should this be tested?
* You may have to uninstall the module and re-install to get the new Views
* View the wiki for additional indexing in Solr instructions. Essentially you're adding the following:

```
/usr/local/solr/collection1/conf/schema.xml

<field name="annotation_value" type="text" indexed="true" stored="true" multiValued="true"/>
<field name="annotation_value_render" type="text" indexed="true" stored="true" multiValued="false"/>

<copyField source="annotation_value" dest="annotation_value_render"/>

vi /usr/local/solr/collection1/conf/solrconfig.xml
annotation_value^1 annotation_value_render^1
```

* View the wiki for updates on context (for images, pages you get the View Block for Images, Pages. for audio, video you get the View Block for Audio, Video

# Interested parties
@Natkeeran @MarcusBarnes

### CODE REVIEW

**Comment by kimpham54** [comment]
not sure why the build is failing... let me know and I can fix it

File: includes/views/islandora_web_annotations.views_default.inc [hunk 1]
```diff
@@ -9,58 +9,148 @@
...
+    $handler->display->display_options['sorts']['annotation_rangeTimeStart_sort']['table'] = 'islandora_solr';
+    $handler->display->display_options['sorts']['annotation_rangeTimeStart_sort']['field'] = 'annotation_rangeTimeStart_sort';
+
+    /* Display: Block */
+    $handler = $view->new_display('block', 'Block', 'islandora_annotations_audio_video');
+
+
+    $views['islandora_web_annotation_block_audio_video'] = $view;
+
+
```
Comment (MarcusBarnes) [inline includes/views/islandora_web_annotations.views_default.inc]: Please remove these empty lines.  Only one needed.

File: includes/views/islandora_web_annotations.views_default.inc [hunk 2]
```diff
@@ -9,58 +9,148 @@
...
+    $view->description = '';
+    $view->tag = 'islandora_web_annotations';
+    $view->base_table = 'islandora_solr';
+    $view->human_name = 'Islandora Web Annotation Block - Audio, Video';
+    $view->core = 7;
+    $view->api_version = '3.0';
+    $view->disabled = FALSE; /* Edit this to true to make a default view disabled initially */
+
+
+    /*
```
Comment (MarcusBarnes) [inline includes/views/islandora_web_annotations.views_default.inc]: Please make sure the docblock characters are aligned properly.

File: islandora_web_annotations.module [hunk 3]
```diff
@@ -144,6 +146,8 @@ function islandora_web_annotations_views_api() {
   );
 }
 
+
```
Comment (MarcusBarnes) [inline islandora_web_annotations.module:148]: Please remove extra lines.

**Comment by kimpham54** [comment]
@MarcusBarnes Just added a new commit, added the fgs_ownerID field to views to indicate the author of an annotation

**Comment by MarcusBarnes** [comment]
Tested and appears to work as expected.  Updating the documentation to provide details on how to adjust the view blocks would be useful.

**Pull request merged**